content/manuals/build/building/best-practices.md

```diff
@@ -147,7 +147,7 @@ Limiting each container to one process is a good rule of thumb, but it's not a
 hard and fast rule. For example, not only can containers be
 [spawned with an init process](/manuals/engine/containers/multi-service_container.md),
 some programs might spawn additional processes of their own accord. For
-instance, [Celery](https://docs.celeryproject.org/) can spawn multiple worker
+instance, [Celery](https://docs.celeryq.dev/) can spawn multiple worker
 processes, and [Apache](https://httpd.apache.org/) can create one process per
 request.
 
```

Comment:
There is another stale documentation URL in the [Dockerfile instructions --> ENTRYPOINT](https://github.com/docker/docs/blob/main/content/manuals/build/building/best-practices.md#entrypoint) section.

> This script uses [the `exec` Bash command](https://wiki.bash-hackers.org/commands/builtin/exec) so that the final running application becomes the container's PID 1.

Probably it could be substituted with a Linux manual page for the `exec` Bash command :thought_balloon:

Reply:
Yes indeed, it needs to be changed in https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/reference.md#entrypoint